uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/login.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-module.exports = function (casper) {
+module.exports = function (casper, scenario) {
   var config = require('../site-config');
   var loginFormSelector = 'form#user-login-form';
+  var credentials = config.credentials[scenario.credential || 'admin'];
 
   casper
-    .echo('Logging in before starting...', 'INFO')
+    .then(function () {
+      if (casper.exists('a[href="/user/logout"]')) {
```

Comment:
you are not using the link anymore, so this check is not necessary

Reply:
true, but still we need to check whether logout is necessary, so edited this part of the code.